### PR TITLE
Improve channel operators

### PR DESCRIPTION
**Pull request by Kelvin-M**
NB: I've not tested it currently !

### CODE REVIEW

File: charts/hlf-k8s/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@
 apiVersion: v1
 name: hlf-k8s
 home: https://substra.org/
-version: 1.3.2
+version: 1.3.3
```
Comment (ClementGautier) [inline charts/hlf-k8s/Chart.yaml:18]: Could you bump to 1.4? This introduce a BC break that I'm not very confortable to put in a minor version.

Comment (Kelvin-M) [reply charts/hlf-k8s/Chart.yaml]: Done

File: charts/hlf-k8s/Chart.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@
 apiVersion: v1
 name: hlf-k8s
 home: https://substra.org/
-version: 1.3.2
+version: 1.4
```
Comment (ClementGautier) [inline charts/hlf-k8s/Chart.yaml:18]: Actually it's 1.4.0, because 1.4 is not a string in Yaml terms, sorry about that
```suggestion
version: 1.4.0
```

Comment (Kelvin-M) [reply charts/hlf-k8s/Chart.yaml]: Redone :)

**Pull request merged**